COLLABORATOR_GUIDE.md

```diff
@@ -294,22 +288,121 @@ After generating one, make sure to keep it safe because GitHub will not show the
 
 ### Doing a release
 
-To do a release, check out the master branch
+It is also recommended you have a clean clone of Video.js for each release line you want to relase.
+That means having a folder for master/v6 and one for 5.x.
+
+```sh
+# for v6
+git clone git@github.com:videojs/video.js.git videojs-6-release
+# for v5
+git clone git@github.com:videojs/video.js.git videojs-5-release
+```
+
+#### Video.js 6
+
+Make sure go to the master branch and grab the latest updates.
 
 ```sh
 git checkout master
+git pull origin master
 ```
 
-Then run the contrib command to do the next release. Don't forget to provide your GitHub token so the GitHub release goes through.
+Then, it's mostly a standard npm package release process with running `npm version`, followed by an `npm publish`.
+
+```sh
+npm version {major|minor|patch}
+```
+
+Depending on the commits that have been merged, you can choose from `major`, `minor`, or `patch` as the versioning values.
+
+Afterwards, you want to push the commit and the tag to the repo.
+It's necessary to do this before running `npm publish` because our GitHub release automation
+relies on the commit being available on GitHub.
 
 ```sh
-VJS_GITHUB_USER=gkatsev VJS_GITHUB_TOKEN=my-personal-access-token contrib release next patch
+git push origin master
 ```
 
-This makes a patch release, you can also do a `minor` and a `major` release.
+Finally, run `npm publish` with an appropriate tag. Don't forget to supply your token.
+
+```sh
+VJS_GITHUB_USER=gkatsev VJS_GITHUB_TOKEN=my-personal-access-token npm publish --tag next
+```
 
 After it's done, verify that the GitHub release has the correct changelog output.
```

Comment:
It would be good to specify that the verification step here is making sure the changelog output isn't garbled